### ISSUE TITLE
Source is too large

### ISSUE THREAD

**Issue by hopewise** [openfl/openfl#2184]
I get this error when I remove the movieClip from stage and add it again using addChild, removeChild, the error occurs when mc has glow-filters:

    openfl 8.9.0
    lime 7.3.0

```
ImageDataUtil.hx:1469 Uncaught RangeError: Source is too large
    at Uint8Array.set (<anonymous>)
    at Function.lime__$internal_graphics_ImageDataUtil.__boxBlur (ImageDataUtil.hx:1469)
    at Function.lime__$internal_graphics_ImageDataUtil.gaussianBlur (ImageDataUtil.hx:664)
    at openfl_filters_GlowFilter.__applyFilter (GlowFilter.hx:236)
    at openfl_display_MovieClip.__updateCacheBitmap (DisplayObject.hx:2519)
    at openfl_display_MovieClip.__renderCanvas (DisplayObject.hx:1721)
    at openfl_display_MovieClip.__renderCanvas (DisplayObjectContainer.hx:963)
    at openfl_display_MovieClip.__renderCanvas (DisplayObjectContainer.hx:973)
    at Led.__renderCanvas (DisplayObjectContainer.hx:973)
    at com_clientside_compoents_Led.__renderCanvas (DisplayObjectContainer.hx:973)
```

**Comment by hopewise** [comment]
I have updated to openfl 8.9.1, lime 7.5.0 and problem is fixed!

**Comment by hopewise** [comment]
The above problem was solved, in lime 7.5.0 however, white glow filter is not showing
